Android/src/main/java/com/applicaster/jwplayerplugin/player/JWPlayerUtil.java

```diff
@@ -140,22 +134,26 @@ public static PlaylistItem getPlaylistItem(Playable playable, Map pluginConfigur
 
     }
 
-    private static List<AdBreak> getPluginConfigurationAdScheduler(Playable playable, Map pluginConfiguration) {
+    public static AdBreak getConfigMidrollInterval() {
+        return postrollAdBreak;
+    }
+
+    public static List<AdBreak> getPluginConfigurationAdScheduler(Playable playable, Map pluginConfiguration) {
 
         // Create your ad schedule
         List<AdBreak> adSchedule = new ArrayList<>();
 
         if (pluginConfiguration != null) {
             if (playable.isLive()) {
                 String liveAdUrl = (String) pluginConfiguration.get("live_ad_url");
-                String liveAdOffset = (String) pluginConfiguration.get("live_ad_offset");
                 String liveAdType = (String) pluginConfiguration.get("live_ad_type");
```

Comment:
Please remove this reference, since we are removing the field from the Config fields